bin/build_cxx

```diff
@@ -15,16 +15,28 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
-set -e
-set -x
-
-top_dir=$(cd $(dirname $(dirname $0)) && pwd)
-
-echo -e "\033[0;32m--- Building cxx sdk ---\n\033[0m"
-cd $top_dir/sdk/cxx
-mkdir -p build
-cd build
-# To build with debug symbols use:
-# cmake -DCMAKE_BUILD_TYPE=Debug ..
-cmake ..
-make
+#set -e
```

Comment:
The changes here have multiple flaws: a) does not properly use top_dir, which then requires the script to be run in a specific directory; b) commented out code.